### DISCUSSION TITLE
Parking app

### DISCUSSION THREAD

**Discussion by mcdowellalex** [decaporg/decap-cms#6755] [Ideas]
This has to be something already. Butt 🍑...

Software system that tracks availability in parking garages. 

This would only be around bigger cities like NY and Chicago. 

But we provide an API and integration for public parking garages that tracks cars that come and go. And an app that allows users to map to a desired location with availability. 

Long term. 
We can provide the ticketing machines that can provide us with more info than just the availability. Like we could reserve spots at a garage if someone books it in advance (plus a fee). 

Probably couldn't give spot numbers. Would need camera work. Potential long long term r&d but feels very out of scope and relevance doesn't feel profitable or beneficial to work on.

**Comment by mcdowellalex** [comment]
@ Toussaint 

Long term idea to work on hardware and embedded systems 👁️👄👁️

**Comment by martinjagodic** [comment]
@mcdowellalex  Wrong repo? 😃

**Reply by mcdowellalex** [reply]
lollll I don't know how that happened, and I'm just noticing
